prereq_flowchart/TEMP_graphviz/test.py

```diff
@@ -46,13 +46,19 @@ def data_graph(data: List[Dict[str, Union[str, List[str]]]]) -> graphviz.Digraph
     """
     retGraph = graphviz.Digraph(
         "wide",
+        edge_attr={
+            "minlen": "2"
+        },
         node_attr={
             "color": "darkgoldenrod1",
             "style": "filled",
             "shape": "invhouse",
             "fontname": "Century Gothic Bold",
         },
     )
+    graphviz.Digraph(graph_attr={
+
+    })
```

Comment:
what does this do?

Reply:
it refreshes the graphviz cache to make sure that generated files are

jk i was fishing with the code completion i'll delete it
```suggestion
```